zebedee-reader/src/test/java/com/github/onsdigital/zebedee/reader/api/bean/PublishedIndexResponseTest.java

```diff
@@ -0,0 +1,84 @@
+package com.github.onsdigital.zebedee.reader.api.bean;
+
+import com.github.onsdigital.zebedee.search.indexing.Document;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.junit.Assert.*;
+
+public class PublishedIndexResponseTest  {
+
+    static final String URI1 = "/uri/1";
+    static final String URI2 = "/uri/2";
+
+    @Test
+    public void testAddDocument_null() {
+        PublishedIndexResponse publishedIndexResponse = new PublishedIndexResponse();
+        publishedIndexResponse.addDocuments(null);
+        assertEquals(publishedIndexResponse.getCount(),0);
```

Comment:
with JUnit the `expected` should come first iirc
```
    public static void assertEquals(Object expected, Object actual)
```

Reply:
Thanks, you're totally right. I've been writing too much Go recently. Will update.